src/player.lua

```diff
@@ -191,18 +191,38 @@ function Player:draw()
 end
 
 function Player:drawWorldObjects()
+	local drawOrder = {
+		"projectiles",
+		"missile",
+		"general",
+		"shield"
+	}
+
+	local fixtureList = {}
+
+	for _, category in ipairs(drawOrder) do
+		fixtureList[PhysicsReferences.getCategory(category)] = {}
```

Comment:
Question: if we made `categories` public so that we could use it to get a list of all the categories, why don't we use that instead of `getCategory`?

```suggestion
		fixtureList[PhysicsReferences.categories[category]] = {}
```

And then get rid of the `getCategory` function.

Reply:
If we do this, then removing `getCategory` should be in a separate commit. No need to slow down this change set for something bigger like that.

Reply:
Agreed